model/app/konnector.go

```diff
@@ -282,7 +282,6 @@ func GetKonnectorBySlugAndUpdate(db prefixer.Prefixer, slug string, copier appfs
 
 // ListKonnectors returns the list of installed konnectors applications.
 //
-// TODO: pagination
 func ListKonnectors(db prefixer.Prefixer) ([]Manifest, error) {
```

Comment:
Why did you keep this function?

Reply:
I was keeping it for the other uses (like intents), but with the same default limit (100), we can safely remove it without risking side-effects